egs/wsj/s5/steps/cleanup/make_utterance_graph.sh

```diff
@@ -105,12 +105,13 @@ while read line; do
       utils/make_unigram_grammar.pl | fstcompile |\
       fstarcsort --sort_type=ilabel > $wdir/G.fst || exit 1;
   else
-    echo $words | awk -v voc=$lang/words.txt -v oov="$oov_txt" '
-      BEGIN{ while((getline<voc)>0) { invoc[$1]=1; } } {
-      for (x=1;x<=NF;x++) {
-      if (invoc[$x]) { printf("%s ", $x); } else { printf("%s ", oov); } }
-      printf("\n"); }' > $wdir/text
-    ngram-count -text $wdir/text -order $ngram_order "$srilm_options" -lm - |\
+     echo $words > $wdir/text
+     split -b 200000 $wdir/text $wdir/xxx
```

Comment:
I don't think this is very ideal because it may split in the middle of a word, leading to some words in the original text not being present in the LM that's produced.  How about this:
```
echo $words | perl -ane '@A = split; for ($n=0;$n<@A;$n++) { print "$A[$n] "; if(($n+1)%2 == 0 || $n+1==@A) {print "\n";} }' > $wdir/text

```